scripts/ci/kubernetes/minikube/_k8s.sh

```diff
@@ -26,15 +27,15 @@ function _wait_for_ready () {
   local evidence="$1"
   shift
   local attempts=40
-  echo "Waiting till ready (count: $count): $@"
-  while [[ "$count" < $("$@" 2>&1 | tail -n +2 | awk '{print $2}' | grep -c $evidence) ]];
+  echo "Waiting till ready (count: ${count}): $*"
+  while [[ "${count}" < $("$@" 2>&1 | tail -n +2 | awk '{print $2}' | grep -c ${evidence}) ]];
   do
     if [[ "$attempts" = "1" ]]; then
-      echo "Last run: $@"
+      echo "Last run: $*"
```

Comment:
Also: why? `$@` is usually better of $# * (it makes no difference here, but why change it?)

Reply:
Not sure any more. Will revert.

Reply:
OK. that was shellcheck-reported problem: https://www.shellcheck.net/wiki/SC2145 - mixing arrays and strings might have unexpected consequences. I kept the change.